akka-docs/src/main/paradox/typed/testing.md

```diff
@@ -11,6 +11,19 @@ To use Akka TestKit Typed, add the module to your project:
   scope=test
 }
 
+@@@div { .group-scala }
+
+We recommend using Akka TestKit Typed with ScalaTest:
+
+@@dependency[sbt,Maven,Gradle] {
+  group=org.scalatest
+  artifact=scalatest_$scala.binary_version$
+  version=$scalatest.version$
+  scope=test
+}
+
+@@@
+
 ## Introduction
 
 Testing can either be done asynchronously using a real `ActorSystem` or synchronously on the testing thread using the `BehaviourTestKit`.
```

Comment:
Needs to change to `BehaviorTestKit`:
```suggestion
Testing can either be done asynchronously using a real `ActorSystem` or synchronously on the testing thread using the `BehaviorTestKit`.
```

Reply:
Good catch! I'm replacing those with `@apidoc[BehaviorTestKit]` so they're checked at HTML generation time and link to the scala/javadoc